apps/fluent-tester/src/RNTester/TestComponents/PersonaCoin/styles.ts

```diff
@@ -1,14 +1,11 @@
-import { StyleSheet } from 'react-native';
+import {StyleSheet} from 'react-native';
```

Comment:
Nit: spacing. It seems like we prefer the spaces?

Reply:
I didn't do this: the `yarn build` did it so I assume its a tslint fix thing

Reply:
I tried to fix it back to the original spacing, but the next `yarn build` put it back to no spaces again.

Reply:
Interesting...